frontend/src/features/permits/components/permit-list/ApplicationInReviewModal.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../../../themes/orbcStyles";
+
+.application-in-review-modal {
+  & &__container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__header {
+    padding: 2rem 1.5rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    background-color: $bc-background-light-grey;
+    border-bottom: 1px solid $bc-border-grey;
+  }
+
+  &__title {
+    font-weight: 600;
+    font-size: 1.5rem;
+    margin: 0 0 0 0.5em;
+  }
+
+  &__body {
+    padding: 1.5rem;
+  }
+
+  &__text {
+    font-size: 1rem;
+  }
+
+  &__footer {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    padding: 0 1.5rem 1.5rem 1.5rem;
+    gap: 1.5rem;
+
+    .remove-users-btn {
+      &--cancel {
+        background-color: $bc-background-light-grey;
+        border: 2px solid transparent;
+        box-shadow: none;
+        color: $bc-black;
+        cursor: pointer;
+
+        &:hover {
+          background-color: $bc-background-light-grey;
+          border-color: $bc-text-box-border-grey;
+          box-shadow: none;
+        }
+      }
+
+      &--confirm {
+        background-color: $bc-red;
+        box-shadow: none;
+        color: $white;
+        cursor: pointer;
+        font-weight: bold;
+
+        &:hover {
+          background-color: $bc-messages-red-text;
+          box-shadow: none;
+        }
+
+        &:disabled {
+          background-color: #EBC0C1;
```

Comment:
Didn't we have any existing disabled colour (in orbcStyles.scss)? Or is this colour newly introduced in the HiFi? If so, we can add it into the orbcStyles.scss as "disabled-secondary-colour" or something, and reused it here.